### ISSUE TITLE
App crashes due to missing resource files

### ISSUE THREAD

**Issue by FelixLisczyk** [liu044100/SmileTouchID#13]
Hi,

I've installed SmileTouchID in my Swift project via CocoaPods and get the following error message upon presenting the AuthViewController:

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Could not find a storyboard named 'SmileSettingVC' in bundle NSBundle </Users/felix/Library/Developer/CoreSimulator/Devices/14EC3B48-C8C2-44EB-99B2-7E6411AD3EB9/data/Containers/Bundle/Application/7868F383-4BAB-448D-BC22-937F73BD2DB5/SmileTouchIDTest.app> (loaded)'
```

It looks like the app cannot find the resource files although they get bundled in the pods target. When I include them manually in my main target, everything runs fine.

I've created a little sample project to demonstrate this issue. Simply run and tap on the button.
[SmileTouchIDTest.zip](https://github.com/liu044100/SmileTouchID/files/148862/SmileTouchIDTest.zip)

Thanks for this great library and your help! ;)

**Comment by liu044100** [comment]
Hello @FelixII , thanks for report this issue😬.

I have fixed it,  call `pod update` and update to `0.1.4` version, it should works.

**Comment by FelixLisczyk** [comment]
Yes that works, thank you very much!
